.github/workflows/manual-nixos.yml

```diff
@@ -0,0 +1,25 @@
+name: "Build NixOS manual"
+
+on:
+  pull_request_target:
+    paths:
+      - 'nixos/**'
+
+jobs:
+  nixos:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # pull_request_target checks out the base branch by default
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - uses: cachix/install-nix-action@v12
+        with:
+          # explicitly enable sandbox
+          extra_nix_config: sandbox = true
```

Comment:
```suggestion
          extra_nix_config: |
            restrict-eval = true
            sandbox = true
```

Reply:
I would not use extra-config here but a full command because it is easier to copy-and-paste when reproducing build errors.